Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -662,6 +666,7 @@
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
 				ASSETCATALOG_COMPILER_GENERATE_SWIFT_ASSET_SYMBOL_EXTENSIONS = YES;
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
 				CLANG_ANALYZER_NONNULL = YES;
```

Comment:
Do you know what the purpose of this setting is? and why it needs to be set to yes?